Modules/_pickle.c

```diff
@@ -6623,13 +6623,9 @@ _pickle_Unpickler_find_class_impl(UnpicklerObject *self,
         }
     }
 
-    module = PyImport_GetModule(module_name);
+    module = PyImport_Import(module_name);
```

Comment:
Could you benchmark how slower this is compared with just using `PyImport_GetModule`?

Reply:
git master:
```
$ ./python -m timeit -s "import pickle, logging; d = pickle.dumps((logging.Logger, logging.Handler, logging.Filter, logging.Formatter))" "pickle.loads(d)"
100000 loops, best of 5: 2.55 usec per loop
```
This PR:
```
$ ./python -m timeit -s "import pickle, logging; d = pickle.dumps((logging.Logger, logging.Handler, logging.Filter, logging.Formatter))" "pickle.loads(d)"
50000 loops, best of 5: 4.84 usec per loop
```

So this is a sizable performance drop, but on a micro-benchmark. I don't think this will be significant in real-world cases. Usually, pickle performance becomes important when large pieces of data are serialized, and by construction you don't get many `STACK_GLOBAL` opcodes in it.

cc'ing @pierreglaser @ogrisel out of curiosity.

Reply:
+1.  Also, `STACK_GLOBAL` should tend to progressively vanish in the pickle string, in favor of `BINGET` as functions/classes get progressively added to the memo (if used).